website/content/docs/v0.14/Cloud Platforms/oracle.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
is there a little draft docs available? Would be great to have some steps, even if it's a draft or manual process